example/properties_directory/non-scalar/hubbard_parameters.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "hubbard_parameters",
+    "units": "eV",
+    "values": [
+        {
+            "category": "hubbard_u",
```

Comment:
The data structure can be simplified if we store U and V separately: remove category